src/test/datascience/notebook/intellisense/completionProvider.vscode.test.ts

```diff
@@ -86,10 +86,17 @@ suite('DataScience - VSCode Notebook - (Code Completion via Jupyter) (slow)', fu
         };
         traceInfo('Get completions in test');
         const completions = await completionProvider.provideCompletionItems(cell2.document, position, token, context);
-        console.log(JSON.stringify(completions));
         const items = completions.map((item) => item.label);
         assert.isOk(items.length);
-        assert.include(items, 'bit_length');
-        assert.include(items, 'to_bytes');
+        assert.ok(
+            items.find((item) =>
+                typeof item === 'string' ? item.includes('bit_length') : item.label.includes('bit_length')
```

Comment:
Fixes tests with failing code completion (this is a bug in jupyter, changes are breaking us, the return value is different)
We already have an issue to fix the underlying bug (this way this test passes)